content/en/sidecar-container.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sidecar Container
+status: Feedback Appreciated
+category: concept
+---
+
+The Sidecar Container is an implementation of the sidecar pattern. 
+An application deployed on a second container will run side by side and share its lifecycle with a primary application running on a main container.
```

Comment:
```suggestion
An application deployed on a second container will run side by side 
and share its lifecycle with a primary application running on a main container.
```

Reply:
Thanks @iamNoah1. I've accepted your changes in commit 38f12ef8614313d91ce8b400be3d99e5857c1964 . I've first made a mistake by committing without the sign-off. Hopefully its resolved.